definitions/install.yaml

```diff
@@ -483,12 +483,87 @@ subjects:
 apiVersion: rbac.authorization.k8s.io/v1beta1
 kind: ClusterRole
 metadata:
-  name: policyviolation
+  name: kyvernoRole
```

Comment:
Would recommend following naming K8s conventions for roles. See: https://kubernetes.io/docs/reference/access-authn-authz/rbac/#default-roles-and-role-bindings.

We can call this role something like  `kyverno:controller`.